Source/Particles/LaserParticleContainer.cpp

```diff
@@ -278,6 +291,12 @@ LaserParticleContainer::UpdateContinuousInjectionPosition (Real dt)
         // which has 3 components, for both 2D and 3D simulations.
         m_updated_position[2*dir] -= WarpX::beta_boost *
             WarpX::boost_direction[2*dir] * PhysConst::c * dt;
+#elif ( AMREX_SPACEDIM == 1 )
+        // In 1D, dir=0 corresponds to z
+        // This needs to be converted in order to index `boost_direction`
+        // which has 3 components, for 1D, 2D, and 3D simulations.
+        m_updated_position[2 + dir] -= WarpX::beta_boost *
```

Comment:
Same comment on `dir` ; could you use `amrex::ignore_unused` here?

Reply:
Done!